social/apps/django_app/default/models.py

```diff
@@ -96,6 +96,9 @@ class Association(models.Model, DjangoAssociationMixin):
 
     class Meta:
         db_table = 'social_auth_association'
+        unique_together = (
```

Comment:
@clintonb , does specifying two fields as `unique_together` also index them together?  I know django provides [`index_together`](https://docs.djangoproject.com/en/1.9/ref/models/options/#index-together), but wasn't sure if unique_together would provide the same functionality